keps/sig-multicluster/4322-cluster-inventory/README.md

```diff
@@ -379,14 +383,37 @@ command or function to work in the same way in another tool. This can
 further enhance the usability and adoption of different cluster
 manager.
 
-### Notes/Constraints/Caveats (Optional)
+### Notes/Constraints/Caveats
+#### What's the relationship between the clusterProfile API and cluster Inventory?
+The clusterProfile API represents a single member cluster in a cluster Inventory.
+
+#### What's the relationship between a cluster Inventory and clusterSet?
+A cluster inventory is considered a clusterSet if all its member clusters adhere to the 
+[namespace sameness](https://github.com/kubernetes/community/blob/master/sig-multicluster/namespace-sameness-position-statement.md) principle.
+
```

Comment:
Do we need to add:

```
#### What's the relationship between a ClusterProfile API and ClusterSet?
```

Reply:
I thought it was transitive as we defined the relationship between ClusterProfile API <-> cluster Inventory and cluster Inventory <-> clusterSet but I think we can be more clear.

Reply:
That still works here I think, given the definition of ClusterInventory to be a collection of ClusterProfile.
A clusterInventory could be taken at the hub-cluster level and still each namespace to be a clusterSet.

I think the ClusterProfile is within a Namespace, then one can consume the inventory at the namespace or hub-cluster level (sum of all namespaces).
For cluster-set, we're implying the namespace level is the way and adding an extra requirement (all clusters in a namespace-level inventory are part of the same clusterset)